tests/fitting/common/test_jobs.py

```diff
@@ -17,7 +17,7 @@ def test_gap_fit_maker(test_dir, memory_jobstore, clean_dir):
         database_dir=database_dir
         )
 
-    responses = run_locally(
+    _ = run_locally(
```

Comment:
why don't we just do `run_locally(...)`?

Reply:
Tests can be made better in future by accessing the responses and assertions.

Reply:
Up to you, just wondered about it 😄